bin/qtl-analysis

```diff
@@ -316,9 +314,12 @@ detDirs=(
 
 # cleanup output directories
 if $enableAna; then
-  if [ -d $finalDirPreQA ]; then
-    printWarning "removing output directory $finalDirPreQA"
-    rm -r $finalDirPreQA
+  ##
+  ## FIXME: is this really necessary?
+  ##
+  if [ -d $finalDir ]; then
+    printWarning "removing output directory $finalDir"
+    rm -r $finalDir
```

Comment:
Hmmm... need to think about this, since it may create some conflicts with `qtl physics`.

Reply:
Things should work okay if either
- `qtl physics` is run _after_ `qtl detectors` (will need to add `--overwrite` to `qtl physics`)
- use a different `dataset` name for the two

I'm not really inclined to make this better, since I'd rather just proceed with refactoring the physics timelines, bringing them into `qtl analysis` territory.